.changeset/spotty-numbers-occur.md

```diff
@@ -0,0 +1,11 @@
+---
+'@backstage/cli': minor
```

Comment:
Would just be a `patch` bump actually, CLI is still on 0.x and this isn't a breaking change